spec/models/plan_spec.rb

```diff
@@ -109,6 +109,22 @@ def plan;
       end
     end
 
+    context "check plan offerings scope" do
+      def params;
```

Comment:
No need to use params / valid_params for this kind of test.

Use FactoryGirl factories and let() to create three plans; is_sole_source, is_vertical, is_horizontal. 

Have three tests that each call the appropriate scope and check that the returned criteria match the created object.

```
let!(:sole_source_plan) { create(:plan, is_sole_source: true) }
...

it "returns only sole source plans" do 
  expect(Plan.check_plan_offerings_for_sole_source.to_a).to match([sole_source_plan])
end
```
something like that